webpack.config.cjs

```diff
@@ -197,6 +197,26 @@ const dashboard = {
   },
 };
 
+const webStoriesScripts = {
+  ...sharedConfig,
+  entry: {
+    lightbox: './assets/src/lightbox/index.js',
+  },
+  plugins: [
+    process.env.BUNDLE_ANALZYER && new BundleAnalyzerPlugin(),
+    new DependencyExtractionWebpackPlugin({
+      injectPolyfill: true,
+    }),
+    new MiniCssExtractPlugin({
+      filename: '../css/[name].css',
+    }),
+    new WebpackBar({
+      name: 'Web Stories Scripts',
```

Comment:
```suggestion
      name: 'Web Stories Lightbox',
```

Reply:
Carousel script will also be added as entry here, so kept it generic `Scripts`. [PR for Carousel](https://github.com/rtCamp/web-stories-wp/pull/201/files?file-filters%5B%5D=.cjs#diff-b724a559a3d863e949957e731407f96e486b45139afbdee10bc1082f5bd229e3R203)